src/wallet/storage.rs

```diff
@@ -157,3 +157,106 @@ impl FileData {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use bip39::Mnemonic;
+    use bitcoin::Network;
+    use bitcoind::tempfile::tempdir;
+    // Generate a dummy mnemonic for testing
+
+    fn generate_dummy_mnemonic() -> String {
+        Mnemonic::generate(12).unwrap().to_string()
+    }
```

Comment:
when you have a single line, no point in putting a function.

Reply:
Initially I thought of adding a couple of things but then reduced it to Mnemonic generator only... Peace, I will change it.